index.html

```diff
@@ -0,0 +1,169 @@
+                        <img src="images/program_icon_02.png" alt="">
+                        <h3>Band</h3>
+                        <p>Big Rock Band, Hollywood, Florida. 3628 likes · 51 talking about this. </p>
+                    </div>
+                    <div class="program_details">
+                        <img src="images/program_icon_04.png" alt="">
+                        <h3>Artist</h3>
+                        <p>Rock music, also called rock and roll, rock & roll, or rock 'n' roll</p>
+                    </div>
+                    <div class="program_details">
+                        <img src="images/program_icon_05.png" alt="">
+                        <h3>band</h3>
+                        <p> Big Rock Band, Hollywood, Florida. 3628 likes · 51 talking about this. </p>
+                    </div>
+                    <div class="program_details">
+                        <img src="images/program_icon_05.png" alt="">
+                        <h3>Artist</h3>
+                        <p> Rock music, also called rock and roll, rock & roll, or rock 'n' roll</p>
+                    </div>
+                </div>
+                <button class="ticket_button">get your concert ticket</button>
+                 <div class="see_whole_program">
+                    <a href="#" class="see_program"> See The Whole Schedule</a>
+                    <div class="line_program_two"></div>
+                 </div>
+
+            </div>
+
+        </section>
+        <section class="features">
+            <div class="features_container">
+                <div class="featured_artists">
+                    <h2>Featured artists</h2>
+                    <div class="line_program"></div>
+                </div>
+                <div class="artist">
+                    <div class="artist_image">
+                        <img src="images/boxes.svg" alt="" class="box_image">
+                        <img src="images/artist_1.webp" alt="" class="normal_image">
+                    </div>
+                    <div class="artist_about">
+                        <h2>Chris Cornell.</h2>
+                        <h4>Known as the king of rock'n'roll</h4>
+                        <p>was born in Mississippi to a poor family. He sang on the radio at age 10. </p>
+                    </div>
+                    
+                </div>
+                <div class="artist">
+                    <div class="artist_image">
+                        <img src="images/boxes.svg" alt="" class="box_image">
+                        <img src="images/artist_3.jpg" alt="" class="normal_image">
+                    </div>
+                    <div class="artist_about">
+                        <h2>Jon Bon Jovi.</h2>
+                        <h4>did in fact invent rock'n'roll</h4>
+                        <p>Charles' proto-soul vocals and Brown's everything-is-a-drum were innovations as profound as Berry's </p>
+                    </div>
+                </div>
```

Comment:
- [ ] It is mentioned in the project requirement that we should use javascript to render the contents of this section dynamically. Kindly remove this part and utilize javascript for the task. 🤓️

**Project requirements**
![image](https://user-images.githubusercontent.com/17243325/217842353-01517771-6949-4e9a-9ae6-bbae50013eb6.png)